acceptance-tests/dsl/src/main/java/org/hyperledger/besu/tests/acceptance/dsl/node/BesuNode.java

```diff
@@ -210,7 +210,10 @@ public BesuNode(
           }
         });
     engineRpcConfiguration.ifPresent(
-        config -> MergeConfigOptions.setMergeEnabled(config.isEnabled()));
+        config -> {
+          MergeConfigOptions.setMergeEnabled(config.isEnabled());
+          MergeConfigOptions.setRollupExtensionEnabled(config.isEnabled());
```

Comment:
It's both in fact. It's an extension because this method is pointless if the `engine_` API is not enabled. 
It has it's own namespace because `engine_` is under a well-defined (and maybe evolving) standard and decided to go with it's own namespace to avoid confusion, because `engine_` can be enabled without rollup. 

Also, if `--Xengine-rollup-extension-enabled=true` then a different Coordinator class instance will be used (extending the merge one)